hardware_accelerators/about-hardware-accelerators.adoc

```diff
@@ -0,0 +1,50 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gpu-accelerators
+
+toc::[]
+
+[IMPORTANT]
+====
+Hardware accelerators are a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
```

Comment:
Is there a reason we are calling this tech preview?  All of our accelerator tools like KMM, NFD and DTK are fully supported.  @egallen thoughts?